applications/osb-portal/src/components/common/OSBSpliButton.tsx

```diff
@@ -43,7 +43,7 @@ export const OSBSplitButton = (props: OSBSplitButtonProps) => {
 
 
     return (
-        <Grid container={true} direction="column" alignItems="center">
+        <Grid id='osb-split-button' container={true} direction="column" alignItems="center">
```

Comment:
A few things to fix here: 
- a potentially reusable component should not define ids, only classes: ids must be guaranteed to be unique
- the name of the component is not really indicative (sorry I missed this in previous pr). Should be more something like OSBApplicationChooserButton